UI/Web/src/app/_services/gestures.service.ts

```diff
@@ -0,0 +1,23 @@
+let swipeCoord = [0, 0];
```

Comment:
This shouldn't be gestures.service as that is for an angular service. This swipe method should be exposed under Utility service.

Reply:
Refactored to expose swipe function under utility.service